### ISSUE TITLE
Possible to release Realworld image SR model?

### ISSUE THREAD

**Issue by AIisCool** [HVision-NKU/SRFormer#19]
Results in paper look very good, would be nice to try it.

**Comment by Z-YuPeng** [comment]
Yes, this is in our plan, and we also planned to make a demo. Due to some other factors that caused the delay, these things will be completed within a month.

**Comment by AIisCool** [comment]
Ok, thank you for quick reply. Looking forward to it.

**Comment by AIisCool** [comment]
Apologies to reopen, but was curious on how progress is going?

**Comment by Z-YuPeng** [comment]
Sorry for the late, we now released a mid-version SRFormer, which is a real-world model. It uses BSRGAN as the downsampling method. This is the link: https://drive.google.com/file/d/1apiX03mDAJkaKzGvdJweOTVwL9aiwdI8/view?usp=sharing
We will also release our real-world SRformer based on Real-ESRGAN downsampling soon.

**Comment by AIisCool** [comment]
@Z-YuPeng Thank you. No worries, please take all the time you need. The BSRGAN downsampling model is very good. Excited for the Real-ESRGAN one too!
